samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,100 @@
-# Bike Sharing Sample: Iteratively Develop and Debug Microservices in Kubernetes
+# Deploy the Bike Sharing sample application to Azure Kubernetes Service
 
-## Quickstart
-[Quickstart: Team development on Kubernetes using Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/quickstart-team-development).
\ No newline at end of file
+BikeSharing is a microservices-based sample application that helps showcase the sandboxing capbilities of [Azure Dev Spaces](https://aka.ms/devspaces). 
+
+Follow the steps below to deploy this sample app to Azure Kubernete Service (AKS).
+
+## Prerequisites
+* [Azure subscription](https://azure.microsoft.com/free)
+* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Helm 2.13 (or greater)](https://github.com/helm/helm/blob/master/docs/install.md)
+
+## Configure Azure resources
+
+1. **Create an Azure Kubernetes Service cluster.** You must create an AKS cluster in a [supported region](https://docs.microsoft.com/azure/dev-spaces/about#supported-regions-and-configurations). The below commands create a resource group called `bikesharing-group` and an AKS cluster called `bikesharing-cluster` in the `eastus` region.
```

Comment:
Is it worth calling out that you need to log into the azure CLI and pick your subscription?  It's a little pedantic but may be useful for someone not used to Azure at all.

Reply:
Agreed, added `az login` step.